app/views/home/viewFinalResults.scala.html

```diff
@@ -29,27 +29,27 @@
             <div class="inner-block-padr">
                 <div class="hgroup">
                     <h1 class="heading-xlarge">Home</h1>
-                    <p class="subtitle">@{cachedUsr.user.firstName + " " + cachedUsr.user.lastName}</p>
+                    <p class="subtitle">@{cachedUser.user.firstName + " " + cachedUser.user.lastName}</p>
                 </div>
             </div>
         </div>
-        @if(WithdrawApplicationRole.isAuthorized(cachedUsr)){
         <div class="grid grid-1-2">
             <div class="page-link">
-                <p class="small-btm-margin">
+                <p data-hideifwithdrawn class="small-btm-margin">
                     <a class="" id="view-application" href="@routes.PreviewApplicationController.present()">View your submitted application</a>
                 </p>
-                <p class="small-btm-margin">
+                @if(WithdrawApplicationRole.isAuthorized(cachedUser)){
+                <p data-hideifwithdrawn data-hideafterparity>
                     <a class="" id="edit-personal-details" href="@routes.PersonalDetailsController.present()">Edit your personal details</a>
                 </p>
+                }
             </div>
         </div>
-        }
     </div>
 
-    @withdrawInformation(cachedUsr)
+    @withdrawInformation(cachedUser)
 
-    <section class="section-border @if(WithdrawnApplicationRole.isAuthorized(cachedUsr)){disabled}" id="mainProgressPage">
+    <section class="section-border" id="mainProgressPage">
```

Comment:
Why is this check removed @if(WithdrawnApplicationRole.isAuthorized(cachedUsr)){disabled} ...
I doubt if there is any java script already doing that ... Please check once